lms/envs/production.py

```diff
@@ -920,6 +920,11 @@ def get_env_setting(setting):
 # Allow extra middleware classes to be added to the app through configuration.
 MIDDLEWARE.extend(ENV_TOKENS.get('EXTRA_MIDDLEWARE_CLASSES', []))
 
+############### Settings for django-fernet-fields ##################
+VEDA_FERNET_KEYS = AUTH_TOKENS.get('VEDA_FERNET_KEYS', [])
+FERNET_KEYS = AUTH_TOKENS.get('FERNET_KEYS', FERNET_KEYS)
+FERNET_KEYS += VEDA_FERNET_KEYS
+
```

Comment:
@feanil I noticed during rebase that you recently removed some settings from platform https://github.com/edx/edx-platform/pull/23756

I want to concatenate fernet keys to be able to decrypt some recently [moved data from veda to val.](https://openedx.atlassian.net/browse/DOS-870). It there some other preferred way to do it or should I go ahead and merge this PR.